david-lafontant/my_blog_app
# Feature/validations spec n+1 problem

### My Blog App : validations, model specs, and n+1 problems

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure the project implements the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n+1.md#:~:text=Project%20requirements,in%20your%20app.)
